### PR TITLE
feat(atomic): add new facet show more/less buttons 

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-698

Screen Shot 2021-06-08 at 4 27 49 PM

### CODE REVIEW

File: packages/atomic/src/locales.json [hunk 1]
```diff
@@ -131,6 +131,14 @@
     "zh-cn": "显示较少的值",
     "zh-tw": "顯示較少的值"
   },
+  "showLessFacetResults": {
+    "en": "Show fewer results for {{label}} facet",
```
Comment (dlutzCoveo) [inline packages/atomic/src/locales.json:135]:
```suggestion
    "en": "Show fewer results for the {{label}} facet",
```

**Review by btaillon — APPROVED** [review]

**Review by nbuissoncoveo — APPROVED** [review]
looks good to me

File: packages/atomic/src/locales.json [hunk 2]
```diff
@@ -131,6 +131,14 @@
     "zh-cn": "显示较少的值",
     "zh-tw": "顯示較少的值"
   },
+  "showLessFacetResults": {
+    "en": "Show fewer results for the {{label}} facet",
```
Comment (samisayegh) [inline packages/atomic/src/locales.json:135]: As far as terminology, should we use facet results or facet values? e.g. `Show fewer values ...`

**Review by samisayegh — APPROVED** [review]

File: packages/atomic/src/locales.json (hunk 2 above)
Comment (ThibodeauJF) [reply packages/atomic/src/locales.json:135]: Good point,  that was from the JSUI, I'll change it in the upcoming PR

**Pull request merged**